applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -288,10 +294,21 @@ static void button_msg_sub_thread(void)
 				LOG_DBG("Play/pause not supported in walkie-talkie mode");
 				return;
 			}
-			/* Starts/pauses the audio stream */
-			ret = le_audio_play_pause();
-			if (ret) {
-				LOG_WRN("Could not play/pause");
+
+			if (strm_state == STATE_STREAMING) {
+				ret = bt_cont_ctrl_pause();
+				if (ret) {
+					LOG_WRN("Could not pause");
+				}
+
```

Comment:
Usually I agree, but when there is an else if on the line below it's easier to read with the newline.

Reply:
Are you following any coding style guidelines that mentions this? 

IIRC Zephyr does not have an official style for this, but it's mostly common to keep empty line after `}` _unless_ the next line starts with a `}` (as Erik suggests).

Reply:
I still think there should be no newlines here. This would then be the only place in the application we do it. Other places have more code than this as well inside the statements.
If you want to improve readability it is better to increase the tab size in your IDE.

Reply:
I'll change it. , I already have 8 spaces for a tab in my IDE. And for future note this is not something we should spend this much time on. It is not worth our time